### DISCUSSION TITLE
How to insert an image in the context page text blocks?

### DISCUSSION THREAD

**Discussion by GerlofdeHaan** [alkem-io/alkemio#651] [Q&A] [answered]
How do I insert an image in the context pages text blocks (see screenshot)? The image button doesn't work as expected.
(I have already tried tot change the example URL).

![afbeelding](https://user-images.githubusercontent.com/102361398/171843606-1728a92b-eda6-410d-93a3-9f4226d53711.png)

**Comment by techsmyth** [comment, marked as answer]
Hi Gerlof, 

Nice to see you using this setup. What you are doing should work, so we may need to look  deeper.

For example the markdown used on https://hub.alkem.io/challengecentric is: 

===
Within this Hub we explore several Challenges related to working Challenge Centric. We want to understand the User and how we can design our platform to facilitate this new way of working.
![](https://i.imgur.com/uUw4yOX.png)
For these Challenges we will for example follow a Design Thinking method (*see image above*) with multiple iterations under [Opportunities](https://hub.alkem.io/challengecentric/challenges/contribute/opportunities), Share knowledge under [Contribute](https://hub.alkem.io/challengecentric/contribute), and share the results under [Canvases](https://hub.alkem.io/challengecentric/challenges/structure/canvases) (such as the example below).
![](https://i.imgur.com/6oBid3d.png) 

Have a look and share your thoughts!

======
Can you please provide the opportunity / challenge that you were editing and the text / image that you want to see? Then we can also try it out. 

@Comoque1 can you please have a look and see if there is perhaps an issue on either the background field display?

Cheers, Neil

**Reply by GerlofdeHaan** [reply]
Hi Neil,

I was editing https://hub.alkem.io/admin/hubs/digileefomgeving/challenges/t3d/opportunities/3dinwinnen/context
And trying to get the following image inserted: https://gemeentelijkgeoberaad.pleio.nl/files/view/3ca145e0-afab-4a79-885e-a4e1d1b5f97c/totaal-3d.jpg (just for a test).

**Reply by Comoque1** [reply]
Hi Gerlof,

I have tried inserting the image from the provided example URL(https://gemeentelijkgeoberaad.pleio.nl/files/view/3ca145e0-afab-4a79-885e-a4e1d1b5f97c/totaal-3d.jpg) in multiple markdowns (ZenHub, Alkemio updates MD and online markdown editor (https://dillinger.io/)) and none of them is rendering the image. 

Would you like to provide another image or same the same image hosted on different location?

Regards,
Evgeni

**Reply by GerlofdeHaan** [reply]
Ah, then the url or the server is the problem? I think it's because the url leads directly to a downloadfile in a Pleio website? 
I just solved it myself by copying the same url in a different way: https://gemeentelijkgeoberaad.pleio.nl/file/download/3ca145e0-afab-4a79-885e-a4e1d1b5f97c/totaal-3d.jpg. 
Like this:
When I select the image, then open it in a new tab in the browser (using the option: "afbeelding openen in een nieuw tabblad") and copy the url of that tab. It works like thad. It's the same url, so can't figure out what's the exact issue here.

![afbeelding](https://user-images.githubusercontent.com/102361398/172391251-588bf44e-e592-4dba-8a4c-37490821efa5.png)

**Reply by techsmyth** [reply]
Hi Gerlof, 

great that worked :)

Yes it looks like the URL you were originally trying to use has a different path ("files") and maybe that has some redirects etc behind it.

Opportunity looks nice! I notice you chose for another image in the end?

Cheers, 

Neil

**Reply by GerlofdeHaan** [reply]
Hi Neil,

Thanks and you are right: it's the "s" that makes the difference :-)
The first image was only for testing. This is the right one.
